validator/node/node.go

```diff
@@ -434,7 +434,9 @@ func (c *ValidatorClient) registerValidatorService(
 		WalletInitializedFeed:      c.walletInitialized,
 		GraffitiStruct:             gStruct,
 		LogDutyCountDown:           c.cliCtx.Bool(flags.EnableDutyCountDown.Name),
-		PandoraService:             pandoraService,
+		// Vanguard: pandora service and vanguard node flag are needed for vanguard chain
+		PandoraService:     pandoraService,
+		EnableVanguardNode: c.cliCtx.Bool(cmd.VanguardNetwork.Name),
```

Comment:
As @blazejkrzak has mentioned I would prefer use for example `VanguardModeEnabled`